megatron/text_generation_server.py

```diff
@@ -164,48 +179,60 @@ def put(self):
             if len(prompts) > 1:
                 return "When doing beam_search, batch size must be 1"
 
-        stop_token=50256
+        stop_token = 3
```

Comment:
Setting this to a constant 3 is wrong. I haven't tested generation yet, but I guess this should be the ID of the EOD token, so we need to get it from the tokenizer.